File: displayview/src/main/java/com/renhejia/robot/display/manager/ClockConfigConst.java
/**
 * 
 */
package com.renhejia.robot.display.manager;

/**
 *
 * @author liujunbin
 */
public interface ClockConfigConst {
	/**
	 * 初始化状态判断
	 */
	public static final String IS_ROBOT_CONFIG_INIT = "is_watch_config_init";
	public static boolean IS_WATCH_CONFIG_INIT_DONE = true;
	public static boolean IS_WATCH_CONFIG_INIT_UNDONE = false;


	public static final String KEY_LTP_CLOCK_SKIN = "ltp_clock_skin";
	public static final String KEY_LTP_CLOCK_SKIN_PATH = "ltp_clock_skin_path";

	public static final String KEY_IS_LONG_PRESS_IN_EDIT = "is_long_press_in_edit";
	public static boolean VALUE_IS_LONG_PRESS_IN_EDIT_TRUE = true;
	public static boolean VALUE_IS_LONG_PRESS_IN_EDIT_FALSE = false;


	public static final String KEY_WATCH_HEIGHT = "watch_height";
	public static int VALUE_WATCH_HEIGHT_DEFAULT = 240;

	public static final String KEY_WATCH_WIDTH= "watch_width";
	public static int VALUE_WATCH_WIDTH_DEFAULT  = 240;


	public static final String KEY_CLOCK_HEIGHT = "clock_height";
	public static int VALUE_CLOCK_HEIGHT_DEFAULT = 0;

	public static final String KEY_CLOCK_WIDTH= "clock_width";
	public static int VALUE_CLOCK_WIDTH_DEFAULT  = 0;

	public static final String KEY_WAETHER_UPDATE_TIME = "weather_update_time";

	public static final String KEY_IS_SHOW_DATE= "is_show_date";
	public static final String KEY_IS_SHOW_WEATHER= "is_show_weather";
	public static final String KEY_IS_RANDOM_CHANGE= "is_random_change";
	public static final String KEY_IS_SHOW_CUSTOM= "is_show_custom";
	public static int VALUE_IS_SHOW_OPENED  = 1;
	public static int VALUE_IS_SHOW_CLOSED  = 0;
	public static final String KEY_CUSTOM_SKIN_URL= "custom_skin_url";
	public static final String KEY_CUSTOM_SKIN_NAME= "custom_skin_name";
	public static final String KEY_TIME_ZONE= "time_zone";
	public static final String KEY_TEMP_MODE= "time_mode";
	public static final String KEY_TEMP_MODE_CEL= "cel";

}

File: displayview/src/main/java/com/renhejia/robot/display/manager/ClockConfigConst.kt
/**
 *
 */
package com.renhejia.robot.display.manager

/**
 *
 * @author liujunbin
 */
interface ClockConfigConst {
    companion object {
        /**
         * 初始化状态判断
         */
        const val IS_ROBOT_CONFIG_INIT: String = "is_watch_config_init"
        const val IS_WATCH_CONFIG_INIT_DONE: Boolean = true
        const val IS_WATCH_CONFIG_INIT_UNDONE: Boolean = false


        const val KEY_LTP_CLOCK_SKIN: String = "ltp_clock_skin"
        const val KEY_LTP_CLOCK_SKIN_PATH: String = "ltp_clock_skin_path"

        const val KEY_IS_LONG_PRESS_IN_EDIT: String = "is_long_press_in_edit"
        const val VALUE_IS_LONG_PRESS_IN_EDIT_TRUE: Boolean = true
        const val VALUE_IS_LONG_PRESS_IN_EDIT_FALSE: Boolean = false


        const val KEY_WATCH_HEIGHT: String = "watch_height"
        const val VALUE_WATCH_HEIGHT_DEFAULT: Int = 240

        const val KEY_WATCH_WIDTH: String = "watch_width"
        const val VALUE_WATCH_WIDTH_DEFAULT: Int = 240


        const val KEY_CLOCK_HEIGHT: String = "clock_height"
        const val VALUE_CLOCK_HEIGHT_DEFAULT: Int = 0

        const val KEY_CLOCK_WIDTH: String = "clock_width"
        const val VALUE_CLOCK_WIDTH_DEFAULT: Int = 0

        const val KEY_WAETHER_UPDATE_TIME: String = "weather_update_time"

        const val KEY_IS_SHOW_DATE: String = "is_show_date"
        const val KEY_IS_SHOW_WEATHER: String = "is_show_weather"
        const val KEY_IS_RANDOM_CHANGE: String = "is_random_change"
        const val KEY_IS_SHOW_CUSTOM: String = "is_show_custom"
        const val VALUE_IS_SHOW_OPENED: Int = 1
        const val VALUE_IS_SHOW_CLOSED: Int = 0
        const val KEY_CUSTOM_SKIN_URL: String = "custom_skin_url"
        const val KEY_CUSTOM_SKIN_NAME: String = "custom_skin_name"
        const val KEY_TIME_ZONE: String = "time_zone"
        const val KEY_TEMP_MODE: String = "time_mode"
        const val KEY_TEMP_MODE_CEL: String = "cel"
    }
}


